File: Java/JavaApplication1/src/String/String1.java

package String;

public class String1 {
    
    public static void main(String[] args) {
        
        String s1= "JAAsia ANjum", s2= "jaasia anjum";
        
        System.out.println("String Size : "+s1.length());
        
        if(s1==s2)
            System.out.println("Equal for =");
        else
            System.out.println("Not Equal for =");
        
        if(s1.equals(s2))
            System.out.println("Equal for .equals");
        else
            System.out.println("Not Equal for .equals");
        
        if(s1.equalsIgnoreCase(s2))
            System.out.println("Equal for Ignore case");
        else
            System.out.println("Not Equal for Ignore case");
        
        boolean contain = s1.contains("ANjum");
        System.out.println("Contains ANjum : "+contain);
        
    }
}

File: Subin Sir/String1.c
#include<stdio.h>

int main()
{
    char name[] = "Jaasia Anjum";
    int i=0;

    printf("%s\n",name);

    while(name[i]!='\0')
    {
        if(name[i]>=97 && name[i]<=122)
        {
            name[i] = 65 + (name[i]- 97);
        }

        else if(name[i]>=65 && name[i]<=90)
        {
            name[i] = 97 + (name[i]- 65);
        }

        i++;
    }

    printf("%s\n",name);

    return 0;
}
